### PR TITLE
readme.md: added benchmarks section

### PR DESCRIPTION
**Pull request by fbarotov**
not linking to the related issue, since it may require much more updates and thus, may not necessarily meet the expectations of readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -277,6 +294,30 @@ ERROR "cp s3://somebucket/file.txt file.txt": object already exists
...
     }
 ```
+## Benchmarks
+Some benchmarks regarding the performance of `s5cmd` are introduced below. For more
+details refer to this [post](https://medium.com/@joshua_robinson/s5cmd-for-high-performance-object-storage-7071352cc09d)
+which is the source of the benchmarks to be presented.
+
+*Upload/download of single large file*
+
+![get/put performance graph](https://miro.medium.com/max/770/0*aWOwrmXJN9XXUHf9)
```
Comment (aykutfarsak) [inline README.md:304]: Could we put the images to s5cmd repository in order not to be affected by the deletion of them from the source?

Comment (fbarotov) [reply README.md:304]: nice point, thanks.

**Comment by igungor** [comment]
rebase please.

**Comment by sonmezonur** [comment]
can we resize benchmark images and make them a little bit smaller? :D

**Pull request merged**